extensions/jdbc/jdbc-postgresql/deployment/src/test/java/io/quarkus/jdbc/postgresql/deployment/DevServicesPostgresqlDatasourceTestCase.java

```diff
@@ -48,7 +48,7 @@ public void testDatasource() throws Exception {
             fail("Datasource should not be null");
         }
         assertTrue(configuration.connectionFactoryConfiguration().jdbcUrl().contains("jdbc:postgresql:"));
-        assertEquals("quarkus", configuration.connectionFactoryConfiguration().principal().getName());
+        assertEquals("quarku_1", configuration.connectionFactoryConfiguration().principal().getName());
```

Comment:
Can't you use the constants? BTW, this is a consequence of the mix you made between username and password. This should be the username here.

Reply:
This would require adding a dependency.